### PR TITLE
Improve english translation

### LINKED ISSUE #128: Localisation issue
The default index page has Chinese character left in English environment. 

`<i class="mdui-icon material-icons seaicon">&#xe8b6;</i> 搜索什么...`

This causes Google Search result shows combination of Chinese character and English. 
Screen Shot 2020-08-28 at 10 23 08

### PR DESCRIPTION
**Pull request by evayde**
- Fixes non-translated strings in the English version
- Also could fix #128 
I think the translation just needed to be rebuild.

### CODE REVIEW

**Comment by yrccondor** [comment]
Oops, you are translating MDx V2, which is not ready to release yet. I'll merge this PR when V2 is ready (will be soon).

File: languages/en_US.po [hunk 1]
```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
...
+msgid ""
+"自 2.0.0 版本起，MDx 已不再依赖 jQuery 且不再在前端载入 jQuery 脚本。如果你在升级 MDx 后碰到页面显示不正常等问题，"
+"请尝试打开此选项。<br><strong>这是一个临时选项，将会在未来版本中被移除。如果你的前端页面中有依赖 jQuery 的其他资源，"
+"请确保其不受 MDx 移除 jQuery 的影响。</strong><br>有关更多从 MDx 1.x 升级至 2.x 的信息，请参阅<a href=\"https://doc.flyhigher.top/mdx/zh-CN/upgrade_tip/\">主题文档</a>"
+msgstr "" 
+"Since version 2.0.0, MDx no longer depends on jQuery and no longer loads jQuery scripts on the front end. "
+"If you encounter problems such as abnormal page display after upgrading MDx, please try to enable this option.<br>"
+"<strong>This is a temporary option and will be removed in a future version. If your front-end page has other resources "
+"that rely on jQuery, please make sure that they are not affected by MDx's removal of jQuery.</strong>"
+"For more information about upgrading from MDx 1.x to 2.x, please refer to the <a href=\"https://doc.flyhigher.top/mdx/zh-CN/upgrade_tip/\">theme document</a>"
```
Comment (yechs) [inline languages/en_US.po:2125]: IMHO, `documentation` fits better into this context than simply `document`

File: languages/en_US.po [hunk 2]
```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
...
+msgstr "Return to top on click"
+
+#: admin_style.php:432
+msgid "开启后，点击标题栏可以返回页面顶部。此设置影响所有页面。</strong>"
+msgstr "Click the title bar after opening to return to top."
+
+#: admin_style.php:502
+msgid "如使用自定义 API，你需要在括号内填写 API URL 并确保 API 返回 <code>{\"text\":\"句子\"}</code> 的 JSON 格式。<a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">详细信息</a></code>"
+msgstr "If you are using a custom API, you'll need to fill-in the API URL in parentheses and make sure that the API returns the JSON format <code>{\"text\":\"quote\"}</code> <a href=\"https://doc.flyhigher.top/mdx/zh-CN/config/custom_api/\" target=\"_blank\">details</a></code>"
+
```
Comment (yechs) [inline languages/en_US.po:2010]: BTW @yrccondor the link's broken

File: languages/en_US.po [hunk 3]
```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
...
+msgid "杂项"
+msgstr "Misc"
+
+#: admin_fn.php:92
+msgid "文章"
+msgstr "Article"
+
+#: 
+msgid "首页推荐文章获取方式"
+msgstr "get recommended articles from"
```
Comment (yechs) [inline languages/en_US.po:2033]: I'm thinking about whether `posts` or `articles` are more suitable in this context

File: languages/en_US.po [hunk 4]
```diff
@@ -1968,6 +1970,160 @@ msgstr "Previous"
...
+msgid "在此自定义“发送评论”按钮上的文本。"
+msgstr "Customize the text on the <q>Send Comment</q> button."
+
+#: 
+msgid "无障碍"
+msgstr "Accessibility"
+
+#: 
+msgid "广告"
+msgstr "Advertising"
```
Comment (yechs) [inline languages/en_US.po:2073]: Personally, I'd prefer to use `advertisements`, or `ads` in short. 
As `advertising` are more commonly used when we are selling our own products, while in this case we are including ads from other people selling their products on our website.
Correct me if I'm wrong, English is not my native language : )

**Review by yechs — COMMENTED** [review]
Hi! Original English translator here. Thanks a lot for contributing!

I've posted some of my thoughts in the review, and would love to discuss them with you and @yrccondor!

**Comment by yechs** [comment]
BTW @yrccondor do we still have any pending strings to add into v2?

File: languages/en_US.po (hunk 2 above)
Comment (yrccondor) [reply languages/en_US.po:2010]: Actually the page is not published yet.

**Comment by yrccondor** [comment]
> BTW @yrccondor do we still have any pending strings to add into v2?

Yes.

The development has been stalled for some time, but I'm back for developing. Hopefully v2 will be released in a few weeks.

**Comment by evayde** [comment]
@yechs Hey, nice to meet you 很高兴认识你. 
I have considered your suggestions in my fix. You were, of course, right. English is also not my first language and my Chinese is pretty bad.
I hope the new commit is better now. I would work on a german translation when the next major release is out.

**Comment by yechs** [comment]
@evayde Vielen Dank! Looking forward to your German translation!

**Review by yrccondor — APPROVED** [review]

**Pull request merged**